tools/archive_metadata_validator/schema/archive_metadata.yaml

```diff
@@ -0,0 +1,109 @@
+# Schema definition for archived study metadata.
+#
+# Rx was used for schema definition and verification (see http://rx.codesimply.com/)
```

Comment:
Never saw rx. It seems to be a project from 2008 without any recent development. Although I like how it looks more than eg json schema, I wonder why rx with custom implementation and not some existing other and wisely used ways to prescribe schema and existing tools to validate? Eg https://json-schema.org/ ?

Reply:
Yes, you are right, it is quite old. I used it some time ago, so it was a quick proof-of-concept.

JSON Schema is definitely the way to go. I Will take a look at jsonschema as replacement

Reply:
FWIW here is our DATS inspired schema(s) for DANDI datasets: https://github.com/dandi/dandi-core/tree/master/assets/schema . You can see the records rendered on datasets pages like https://gui.dandiarchive.org/#/dandiset/5e6eb2b776569eb93f451f8d .  Strong accent for authors' records is made to just make them specify their ORCID ID.  So then (AFAIK) we would just fetch the rest from ORCID service

Reply:
Replaced Rx with JSON Schema and added an ORCID ID, which is a brilliant idea.

Thanks for the pointer to DANDI dataset schemata. I (will) shamelessly re-use the idea of controlled vocabulary for roles, once the requirements for the study archival are clearer

Reply:
Great! More reuse/alignment - the better. Yet you rereview changes, Note meanwhile: https://github.com/dandi/dandiarchive/issues/360 (I believe @satra forwarded those slides to @mih ;-))